exercises/ansible_rhel/1.1-setup/README.md

```diff
@@ -19,13 +19,13 @@ Login to your control host via SSH:
 
 > **Warning**
 >
-> Replace **11.22.33.44** by the **IP** provided to you, and the **X** in student**X** by the student number provided to you.
+> Replace **11.22.33.44** by your **IP** provided to you, and the **X** in student**X** by the student number provided to you.
 
     ssh studentX@11.22.33.44
 
 > **Tip**
 >
-> Use the password provided on the workshops landing page.
+> The password is **ansible**
```

Comment:
the password changes based on every workshop, i can't accept this PR because of this single line here

Reply:
I didn't intend to include the 6 other commits.

Reply:
For example, line 22 in this commit is not grammatically correct.

Reply:
ok can you fix the PR?  This is not something I do on my end, you have to update your PR